doc/src/drv_arch/drv_arch.md

```diff
@@ -82,6 +82,15 @@ user-space before calling the reconfiguration IOCTL.
 
 1. Unload the driver from the guest
 2. Unplug the VF from the guest
+
+.. note::
+
+```
+    NOTE: Unplugging the VF from the guest while an application on the guest is
```

Comment:
Is it necessary to include `NOTE:` even though it' under the `.. note::` markup? I don't know how it will look once rendered.

Reply:
I don't know - I'm playing it safe and am sticking with the markup that's already there.

Looking at the current output at https://opae.github.io/0.13.0/docs/drv_arch/drv_arch.html, it doesn't seem to make a difference, though.